snap/local/archivecleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Need to be root to switch to snap _daemon_ user."
+  exit
+fi
```

Comment:
Please use https://google.github.io/styleguide/shellguide.html
```suggestion
if [ "${EUID}" != "0" ]; then
  echo "Error: run it as root (to utilize snap user _daemon_)." >&2
  exit 1
fi
```

1) Errors should go to STDERR , with proper exit code
2) [ "${EUID}" -ne 0 ] compares string to integer using integer operator. Either compare strings with "=" or integers with "-ne". Mixing will cause issues. In this particular case I recommend stings as EUID can be undefined and comparing integers will cause:
```
11:32:02 ✔ taurus:~$ if [ $BLA -ne 0 ]; then echo FOO; fi
bash: [: -ne: unary operator expected
```


Please correct it here and below.